code/datums/mutations.dm

```diff
@@ -583,32 +591,47 @@
 		message = replacetext(message," muh valids "," getting my kicks ")
 	return trim(message)
 
-
 /datum/mutation/human/laser_eyes
-
 	name = "Laser Eyes"
 	quality = POSITIVE
 	dna_block = NON_SCANNABLE
 	text_gain_indication = "<span class='notice'>You feel pressure building up behind your eyes.</span>"
+	above_body = 1
+
+/datum/mutation/human/laser_eyes/New()
+	..()
+	visual_indicators |= image("icon"='icons/effects/genetics.dmi', "icon_state"="lasereyes_s", "layer"=-FRONT_MUTATIONS_LAYER)
+
+/datum/mutation/human/laser_eyes/get_visual_indicator(mob/living/carbon/human/owner)
+	return visual_indicators[1]
 
 /datum/mutation/human/laser_eyes/on_ranged_attack(mob/living/carbon/human/owner, atom/target)
 	if(owner.a_intent == "harm")
 		owner.LaserEyes(target)
 
+/mob/living/carbon/proc/update_mutations_overlay()
+	return
 
-/mob/living/carbon/human/proc/update_mutation_overlays()
-	remove_overlay(MUTATIONS_LAYER)
-	var/image/standing
+/mob/living/carbon/human/update_mutations_overlay()
```

Comment:
Im not sure i like this "rewrite all mutations every time you get new one or lose old one"
If you have a list with all the images stored already why not to just substract that list from the overlays_standing[neededlayer] list and then update overlays?